src/main/java/seedu/foodiebot/model/food/UniqueFoodList.java

```diff
@@ -39,28 +40,23 @@ public void add(Food toAdd) {
         internalList.add(toAdd);
     }
 
-    /**
-     * Replaces the canteen {@code target} in the list with {@code editedPerson}. {@code target}
-     * must exist in the list. The canteen identity of {@code editedPerson} must not be the same as
-     * another existing canteen in the list.
-     */
-    public void setFood(Food target, Food editedFood) {
-        requireAllNonNull(target, editedFood);
-
-        int index = internalList.indexOf(target);
-        if (index == -1) {
-            throw new StallNotFoundException();
-        }
-
-        if (!target.isSameFood(editedFood) && contains(editedFood)) {
-            throw new DuplicateStallException();
-        }
-
-        internalList.set(index, editedFood);
-    }
+    //public void setFood(Food target, Food editedFood) {
```

Comment:
Why did you comment all this out?

Reply:
It gave a checkstyle error when creating the pull request. Apparently there is a another setFood function at the bottom of the class, the one I commented out was never used.